transactions/FanTop/initialize_collection.cdc

```diff
@@ -1,16 +1,16 @@
-import FanTopToken from 0xFANTOP_ADDRESS
+- Arguments: []
```

Comment:
please remove `Arguments` and keep the original script.

Reply:
thanks, I fixed it.